src/ui/interaction/files/AvFileUpload/AvFileUpload.vue

```diff
@@ -207,46 +294,44 @@ function onChange ($event: InputEvent) {
     @dragleave="onDragLeave"
     @drop="onDrop"
   >
-    <div
-      class="file-upload-container"
-      :class="{
-        'file-upload-container--error': error,
-        'file-upload-container--valid': validMessage,
-      }"
-    >
-      <div class="file-upload-content">
-        <div class="left-content-container">
-          <slot name="left">
-            <AvVIcon
-              :size="2.5"
-              :name="MDI_ICONS.ATTACHMENT_PLUS"
-              color="var(--icon)"
-            />
-          </slot>
+
+    <AvFileUploadTemplate class="file-upload-container">
+      <template #content>
+        <div
+          v-if="!enableMultiple && !modelValue"
+          class="title-container"
+        >
```

Comment:
```suggestion
        <div class="title-container">
          <div v-if="!enableMultiple && !modelValue">
```
Ton else utilise aussi `title-container` donc autant le définir qu'une fois